FCCSW_ecal/runSlidingWindowAndCaloSim.py

```diff
@@ -146,7 +146,7 @@
 # EM scale calibration (sampling fraction)
 from Configurables import CalibrateInLayersTool
 calibEcalBarrel = CalibrateInLayersTool("CalibrateECalBarrel",
-                                   samplingFraction = [0.3632447480841956] * 1 + [0.13187261040190248] * 1 + [0.14349714292943705] * 1 + [0.150266118277841] * 1 + [0.15502683375826457] * 1 + [0.15954408786354762] * 1 + [0.16375302347299436] * 1 + [0.16840384714588075] * 1 + [0.17219540619311383] * 1 + [0.1755068643940401] * 1 + [0.17816980262822366] * 1 + [0.18131266048670405] * 1,
+                                   samplingFraction=[0.3864252122990472] * 1 + [0.13597644835735828] * 1 + [0.14520427829645913] * 1 + [0.1510076084632846] * 1 + [0.1552347580991012] * 1 + [0.159694330729184] * 1 + [0.1632954482794191] * 1 + [0.16720711037339814] * 1 + [0.17047749048884808] * 1 + [0.17461698117974286] * 1 + [0.1798984163980135] * 1 + [0.17920355117405806] * 1,
```

Comment:
Are those sampling fractions corresponding to `$FCCDETECORS/Detector/DetFCCeeIDEA-LAr/compact/FCCee_DectMaster.xml` ?

Reply:
No, I ran the full chain in https://github.com/giovannimarchiori/LAr_scripts/blob/gmarchio-main-updatescripts-20240212/FCCSW_ecal/run_all_chain.sh with 
runname="thetamodulemerged_topoclusters"
xmlbasedir=../../k4geo
xmldir=FCCee/ALLEGRO/compact/ALLEGRO_o1_v02
xmlfileFullDet=ALLEGRO_o1_v02
xmlfileECal=ECalBarrel_thetamodulemerged

so these SFs are for k4geo/FCCee/ALLEGRO/compact/ALLEGRO_o1_v02/ECalBarrel_thetamodulemerged.xml

Reply:
reverted the changes to these legacy scripts